src/app/components/AssignmentForm.tsx

```diff
@@ -142,67 +153,77 @@ export default class AssignmentForm extends React.Component<AssignmentFormProps
-                                            <Field name={`${recurrenceInfoFieldName}.endTime`} component={DateTimeFieldConst.TimeField} label="End Time" />
-                                            <Field name={`${recurrenceInfoFieldName}.sheriffsRequired`} component={TextField} label="Number of Sheriffs Required" validate={[Validators.required, Validators.integer]} />
-                                        </ListGroupItem>)
+                                            <Button 
+                                                bsStyle="danger" 
+                                                onClick={() => fields.remove(index)} 
+                                                className="pull-right"
+                                            >
+                                                <Glyphicon glyph="trash" />
+                                            </Button>
+                                            <br />
+                                            <Field
+                                                name={`${recurrenceInfoFieldName}.days`}
+                                                component={DaysOfWeekChecklist}
+                                                label="Days"
+                                            />
+                                            <Field
+                                                name={`${recurrenceInfoFieldName}.startTime`}
+                                                component={DateTimeFieldConst.TimeField}
+                                                label="Start Time"
+                                            />
+                                            <Field
+                                                name={`${recurrenceInfoFieldName}.endTime`}
+                                                component={DateTimeFieldConst.TimeField}
+                                                label="End Time"
+                                            />
+                                            <Field
+                                                name={`${recurrenceInfoFieldName}.sheriffsRequired`}
+                                                component={TextField}
+                                                label="Number of Sheriffs Required" 
+                                                validate={[Validators.required, Validators.integer]}
+                                            />
+                                        </ListGroupItem>
+                                    )
                                 }
                                 )}
                                 <br />
-                                <Button onClick={() => fields.push({
-                                    startTime: moment().hour(9).minute(0),
-                                    endTime: moment().hour(17).minute(0)
-                                })} >
+                                <Button 
```

Comment:
Is the field array interface back in play?  I recall a mentioning that we were opting for a simpler form?

Reply:
It is still in play for the assignment form, as far as I know, but we opted for a simpler interface when creating shifts.

Reply:
Ah yes, this is the assignment form.  My mistake.